Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -45,21 +45,61 @@ struct TextureAndTLUTFormat
 struct EFBCopyParams
 {
   EFBCopyParams(PEControl::PixelFormat efb_format_, EFBCopyFormat copy_format_, bool depth_,
-                bool yuv_)
-      : efb_format(efb_format_), copy_format(copy_format_), depth(depth_), yuv(yuv_)
+                bool yuv_, float y_scale_)
+      : efb_format(efb_format_), copy_format(copy_format_), depth(depth_), yuv(yuv_),
+        y_scale(y_scale_)
   {
   }
 
   bool operator<(const EFBCopyParams& rhs) const
   {
-    return std::tie(efb_format, copy_format, depth, yuv) <
-           std::tie(rhs.efb_format, rhs.copy_format, rhs.depth, rhs.yuv);
+    return std::tie(efb_format, copy_format, depth, yuv, y_scale) <
+           std::tie(rhs.efb_format, rhs.copy_format, rhs.depth, rhs.yuv, rhs.y_scale);
   }
 
   PEControl::PixelFormat efb_format;
   EFBCopyFormat copy_format;
   bool depth;
   bool yuv;
+  float y_scale;
+};
+
+struct TextureLookupInformation
+{
+  u32 address;
+
+  unsigned int block_width;
+  unsigned int block_height;
+  u32 bytes_per_block;
+
+  unsigned int expanded_width;
+  unsigned int expanded_height;
+  unsigned int native_width;
+  unsigned int native_height;
+  u32 total_bytes;
+  u32 native_levels = 1;
+  u32 computed_levels;
+
+  u64 base_hash;
+  u64 full_hash;
+
+  TextureAndTLUTFormat full_format;
+  u32 tlut_address = 0;
+
+  bool is_palette_texture = false;
+  u32 palette_size = 0;
+
+  bool use_mipmaps = false;
+
+  bool from_tmem = false;
+  u32 tmem_address_even = 0;
+  u32 tmem_address_odd = 0;
+
+  int texture_cache_safety_color_sample_size = 0; // Default to safe hashing
+
+  u8* src_data;
+
+  std::string dump_base_name = "";
```

Comment:
std::string initializes to empty by default, so it doesn't need an initializer. Also `<string>` needs to be included.